iphone/Classes/TiUIiOSStepper.m

```diff
@@ -0,0 +1,204 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIIOSSTEPPER
+#import "TiUIiOSStepper.h"
+#import "TiUIiOSStepperProxy.h"
+#import "TiUtils.h"
+
+@implementation TiUIiOSStepper
+
+-(UIStepper*)stepper
+{
+    if (stepper ==nil)
+    {
+        stepper = [[[UIStepper alloc]init] retain];
```

Comment:
Missing space in `alloc]init]`. Also, why do we need to manaully retain the UI element? Don't know of a case where it would be nil here.